Documentation/v2/admin_guide.md

```diff
@@ -165,13 +165,16 @@ $ ssh 10.0.1.11
 ```
 
 ```sh
-$ kill `pgrep etcd`
+$ systemctl stop etcd2
 ```
 
 #### Copy the data directory of the now-idle member to the new machine
 
+`%data_dir%` is usually `/var/lib/etcd2`.
+
 ```
-$ tar -cvzf infra1.etcd.tar.gz %data_dir%
+$ cd %data_dir%
+$ tar -cvzf infra1.etcd.tar.gz *
```

Comment:
I think it's cleaner than having a tar that spills in the working directory / having a `--strip-components`. I'm not too worried about a system admin shirking due diligence and somehow clobbering a host this way.